bindinfo/handle.go

```diff
@@ -60,15 +61,26 @@ type BindHandle struct {
 		atomic.Value
 	}
 
+	dropBindRecordMap struct {
```

Comment:
Why just use `sync.Map`?

Reply:
I read tidb's code, it always uses map and lock

Reply:
Golang's `sync.Map` is introduced in Golang 1.9.
There may be no this struct when others wrote these TiDB code.

Reply:
But it's ok, we can replace it in another PR.